### DISCUSSION TITLE
Add last deployed branch (or tag) information in Project Homepage

### DISCUSSION THREAD

**Discussion by frengooo** [mia-platform/community#169] [Ideas]
Hi guys,
often I need to know the last deployed version in the environments of a project.

At the moment I need to navigate to the "Deploy" page and check it for each environment.
Can we add this information to the homepage of the project, like in the following screenshot?
<img width="1302" alt="Screenshot 2023-01-11 alle 10 50 24" src="https://user-images.githubusercontent.com/28152586/211774388-78dcb72e-b8f1-47a8-b6cf-935da83a98f6.png">

**Comment by akelity** [comment]
This feature has been positively evaluated and may be considered for inclusion in the roadmap in the future, depending on priorities and overall specific needs.
